src/admin.ts

```diff
@@ -0,0 +1,289 @@
+import { applyWriteConcern } from './utils';
```

Comment:
this isn't actually a new file, if you check out the source locally and run `git log --follow src/admin.ts` you'll see the whole history from its javascript past.